components/bl-sweet-alert/src/index.js

```diff
@@ -0,0 +1,68 @@
+import { useCallback, useEffect, useMemo, useRef } from 'react';
+
+import Swal from './lib/sweetalert2';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function SweetAlertComponent({ component, eventHandlers, elRef }) {
+  const {
+    style, classList, display, position, title, text, icon, iconColor, footer, grow, backdrop,
+    allowOutsideClick, allowEscapeKey, allowEnterKey, buttonsSettings, additionalSettings,
+  } = component;
+
+  const swalInstanceRef = useRef(null);
```

Comment:
Why do we need swalInstanceRef is we can at any time call `Swal.getPopup()` or something like that to know if popup is exist?